src/sentry-config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  dsn: 'https://eb128ce663d24c3caacc0331f61e9950@sentry.io/1500311',
```

Comment:
We should probably add this value to [src/local-settings.js](https://github.com/aragon/aragon/blob/master/src/local-settings.js), so that only the official builds will have it enabled.

Reply:
Where should the env variable be defined? I'm only finding variables for running it locally at `scripts/launch-local`

Reply:
We should add it in three places in this file:

With the constants at the top:

```js
const SENTRY_DSN = 'SENTRY_DSN'
```

In the `CONFIGURATION_VARS` array (this is needed because `process.env.XYZ` values are statically transformed):

```js
const CONFIGURATION_VARS = [
  // …
  [SENTRY_DSN, process.env.REACT_APP_SENTRY_DSN],
].reduce(
```

And as a getter function:

```js
export function getSentryDsn() {
  return getLocalSetting(SENTRY_DSN, '')
}
```
<hr />

Then we could add it to the `build` script maybe? https://github.com/aragon/aragon/blob/master/scripts/build#L9

Or we could remove it entirely from the codebase, and only pass it when building the app:

`REACT_APP_SENTRY_DSN=https://eb128ce663d24c3caacc0331f61e9950@sentry.io/1500311 npm run build`

And only initialize Sentry if `getSentryDsn()` returns something. That way, if people modify Aragon, it won’t contain our Sentry configuration.

Reply:
Working on it! Thx!